### PR TITLE
Added lobby sound default off option to config

### PR DESCRIPTION
**Pull request by TDSSS**
**What does this PR do:**
There is now a server config to have lobby sounds default to off for new players.
This is mostly intended for use with test servers which you start/stop a lot, where you don't bother also starting a database. Who doesn't know the frantic 'let's turn off the annoying lobby music' effect when debugging?

Comes with a corresponding option in the example config file, too. Also fixed a typo in the preference menu where  it said "LobbyMusic" instead of Lobby Music

**Changelog:**
:cl: 
add: server option for lobby music to default to off for new players
/:cl:

### CODE REVIEW

**Comment by Ty-Omaha** [comment]
remove lobby music all together

File: config/example/config.txt [hunk 1]
```diff
@@ -385,4 +385,7 @@ HIGH_POP_MC_MODE_AMOUNT 65
 DISABLE_HIGH_POP_MC_MODE_AMOUNT 60
 
 ##Uncomment to enable developer start. Auto starts the server after initialization
-##DEVELOPER_EXPRESS_START
\ No newline at end of file
+##DEVELOPER_EXPRESS_START
+
+##Uncomment to have the lobby sound default to off for new players, useful for testing servers without database.
+LOBBY_SOUND_OFF
```
Comment (Birdtalon) [inline config/example/config.txt]: Comment it out as default.

Comment (TDSSS) [reply config/example/config.txt]: woops, forgot to do that when I copied it over from my config.txt

**Comment by KasparoVy** [comment]
Cant hear my lobby music, my regular music is too loud 😎. I like it on the live for the nostalgia but I really wish I could ineddependently adjust its volume.

**Comment by tigercat2000** [comment]
Um, wat. This is already a config option. 
https://github.com/ParadiseSS13/Paradise/blob/master/code/controllers/configuration.dm#L185

https://github.com/ParadiseSS13/Paradise/blob/master/config/example/config.txt#L330-L331

**Comment by Aurorablade** [comment]
yeah its already an config option..and it can be set per a local basis. and dosen't need to be DEFUALT off.

**Comment by TDSSS** [comment]
lmao. Nobody told me that's an option when I brought it up in coder chat, either. Lol, time to close this, then.

**Comment by Birdtalon** [comment]
First I've heard that it's an option lol

**Pull request closed without merging**